.github/workflows/jira_issue_manage.yml

```diff
@@ -26,11 +26,23 @@ jobs:
         run: pip install -r ${SCRIPT_DIR}/requirements.txt
       - name: install pandoc
         run: sudo apt-get update && sudo apt-get install -y pandoc
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_SM_READONLY_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SM_READONLY_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: get secrets from aws sm
+        uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          secret-ids: |
+            ,sdlc/prod/github/jira
```

Comment:
I assume the empty list before the leading comma means "all IDs" or something? Maybe a comment here would be useful.

Reply:
that's just how the plugin is documented to be used for [environmental variable naming with blank aliases](https://github.com/aws-actions/aws-secretsmanager-get-secrets?tab=readme-ov-file#environment-variable-naming):
> If you set `parse-json-secrets: true` and enter a blank alias, followed by a comma and then the secret ID, the action names the environment variable the same as the parsed JSON keys.